examples/golang/topic-creation/README.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+This directory contains Golang code example on how to create an SNS Topic on a Ceph RGW cluster.
+
+# Prerequisite
+Linux machine running an RGW Ceph cluster. Preferrably started with the ``OSD=1 MON=1 MDS=0 MGR=0 RGW=1 ../src/vstart.sh --debug --new `` command.  
+Go installed on the Linux machine.  
```

Comment:
Here you call it Go, above Golang, maybe use one or the other consistently.  Since you use Go below, maybe make Golang above -> Go